frontend/main.js

```diff
@@ -63,6 +63,8 @@ async function startApp () {
     })
   }
 
+  sbp('okTurtles.data/set', 'LIGHTWEIGHT_CLIENT', process.env.LIGHTWEIGHT_CLIENT ?? true)
```

Comment:
No need to provide another fallback value for `process.env.LIGHTWEIGHT_CLIENT` here since one is already specified in the Grunt file.